src/pages/guides/index.md

```diff
@@ -270,7 +271,7 @@ As mentioned earlier, I/O Events transforms your event payload and adds an addit
 Upon receiving a request, you must do the below for leveraging the enhanced security measures
 
 - verify you are the actual recipient of the event using the new `recipient_client_id` field available in the payload
-- once verified, your app should fetch the public key using the cloud front url present in the header
+- once verified, your app should fetch the public key by forming and validating the url using the adobe domain adobeioevents.com and the relative path received from the request header.
```

Comment:
Same above comment is applicable here as well.

Reply:
@fmeschbe : are you talking about the next statement which mentions caching the key? that obviously mentions how long they should cache the keys for.. we can add the missing detail for the cache header which won't be there in the request and the rotation which is just the changing public key file names